packages/components/src/utils/table/table.ts

```diff
@@ -0,0 +1,49 @@
+import type { Row, Table } from '@tanstack/react-table';
+
+export const selectRow = <TData>(table: Table<TData>, rowId?: string) => {
+  if (!rowId || rowId === '') {
+    table.setRowSelection({});
+  } else {
+    table.setRowSelection({ [`${rowId}`]: true });
+  }
+};
+
+export const getFirstSelectedRow = <TData>(table: Table<TData>) => {
+  return getRow(table, Object.keys(table.getState().rowSelection)[0]);
+};
```

Comment:
Not sure why we need to create our own function if tanstack table has something for this out of the box?
`table.getSelectedRowModel().flatRows[0]`